lib/internal/source_map/source_map.js

```diff
@@ -303,8 +303,20 @@ function decodeVLQ(stringCharIterator) {
 
   // Fix the sign.
   const negative = result & 1;
-  result >>= 1;
-  return negative ? -result : result;
+  // Use unsigned right shift, so that the 32nd bit is properly shifted to the
```

Comment:
```suggestion
  // Use logical right shift, so that the 32nd bit is properly shifted to the
```

Reply:
Unsigned right shift is the name of the `>>>` operator. https://tc39.es/ecma262/#sec-unsigned-right-shift-operator